docs/docs/integrations/llms/oci_model_deployment_endpoint.ipynb

```diff
@@ -54,6 +48,34 @@
     "## Example"
```

Comment:
> add more structured examples:

I did this in `ChatModel` but not in here as LC advertise to Chatmodel not text completions in many providers. In addition, since LangChain already have many invocation/async/streaming examples, our doc can keep it simple. User are expected to learn these langchain's functionalities from LangChain's doc. 

In case of difference between using OCIModelDeploymentLLM and OCIModelDeploymentVLLM, I put into the inline comments in code. I haven't put it into details as I need to know which one we want to advertise more. I would prefer we have discussion on documentation's content in a meeting later.

Reply:
I have provided the link for api doc regarding differences between framework specific initialization and generic initialization. I believe in this way can provide the most valuable information for user.